SparkPost/2web2ui
# Templatesv2 routes

This is based on TR-1358. 

Move new template works under `/templatesV2` directory 

This is done for a few reasons including:
1) To avoid naming files like `ListPageV2` 
2) to minimize number of changes required during release (& removal of v1)
3) to avoid unintentionally affecting current template works while working on v2
4) to wire up routes correctly within v2


### What Changed
 - Moved EditAndPreviewPage to `/templatesV2` directory
 - Update routes and remove unused routes
 - Move related components

### How To Test
- Add necessary feature flag - https://github.com/SparkPost/2web2ui/pull/1032
- Make sure existing templates are working fine
- Make sure new template routes (/templatesv2) are working in
 
### Todo

- [x]  Move tests
